manifest.json

```diff
@@ -30,7 +30,7 @@
         "modal.html"
     ],
     "content_scripts": [{
-        "js": ["libs/jquery.min.js", "libs/extLibs.js", "main.js"],
+        "js": ["libs/jquery.min.js", "libs/extLibs.js", "libs/virtual-key-code.min.js", "main.js"],
```

Comment:
I don't encourage or don't like use another libs because a small function as `keyCode`.We just need define a keyCode object is enough